### ISSUE TITLE
Error when using test helper in sortable-objects with handle

### ISSUE THREAD

**Issue by Fl0rianFischer** [mharris717/ember-drag-drop#107]
First of all: great plugin, thanks! I'm using Ember 2.0 and ember-drag-drop 0.4.5

I get an error `TypeError: Cannot read property 'target' of null` when using the drag test helper. It originates in the drag coordinator service but I tracked it down to the `dragReady` flag not being set in the `didInsertElement` hook of draggable-object in the tests when using a drag handle.

**Comment by dgavey** [comment]
Yes I noticed this the other day as well, went to make an issue to fix it.  You can use version 0.4.3 and it won't have this bug. I need to fix this soon. It was introduced with some fixes for mobile.

**Comment by Fl0rianFischer** [comment]
Thanks so much for you quick answer. You saved my day.

**Comment by Fl0rianFischer** [comment]
In version 0.4.3 I'm getting this problem

https://github.com/mharris717/ember-drag-drop/issues/98

**Comment by dgavey** [comment]
Hmm, odd I'm sure it was the same issue.

**Comment by Fl0rianFischer** [comment]
I'm pretty sure you're right. But when using 0.4.3 the drag test helper seems to be unusable due to another helper not being exposed.
